src/Codeception/Lib/Driver/PostgreSql.php

```diff
@@ -164,7 +164,7 @@ public function getPrimaryKey(string $tableName): array
                 FROM   pg_index i
                 JOIN   pg_attribute a ON a.attrelid = i.indrelid
                                      AND a.attnum = ANY(i.indkey)
-                WHERE  i.indrelid = '{$tableName}'::regclass
+                WHERE  i.indrelid = " . $this->postgreQuotedName($tableName) . "::regclass
```

Comment:
We could explicitly quote every table name without the extra overhead of a regex. I believe it's fully backward-compatible. What do you think?

```suggestion
                WHERE  i.indrelid = '\"{$tableName}\"'::regclass
```

Reply:
I created the postgreQuotedName method because I was not sure it would be backwards compatible, but if it works, your suggestion it's ok for me.

Reply:
All tests pass. Changed.